site/overview/getting-started.md

```diff
@@ -20,48 +20,172 @@ head:
 
 # Getting Started
 
-This guide will help you get started with Account Kit by setting up your environment, creating a Light Account (a type of smart account implementation), and sending a User Operation from it. By the end of this guide, you'll have a basic understanding of how to use the SDK and where to look for more advanced use cases.
+This guide will help you get started with Account Kit by setting up your environment, creating a smart account, and sending a `UserOperation` on its behalf. By the end of this guide, you'll have a basic understanding of how to use the SDK and where to look for more advanced use cases.
 
-## Install the Packages
+## 1. Install the Packages
 
-In your project directory, you'll need to install the required packages:
+In your project directory, you'll need to run the following to install the required packages:
 
 ::: code-group
 
 ```bash [npm]
+npm init -y
+npm install -save-dev typescript
 npm install @alchemy/aa-alchemy @alchemy/aa-accounts @alchemy/aa-core viem
 ```
 
 ```bash [yarn]
+yarn init -y
+yarn add -D typescript
 yarn add @alchemy/aa-alchemy @alchemy/aa-accounts @alchemy/aa-core viem
 ```
 
-Note that we're installing [viem](https://viem.sh/) as well. Viem contains helpful abstractions and modules that may come in handy when using Account Kit. Additionally, many Account Kit modules leverage viem themselves.
+::: tip Note
+
+We're installing [viem](https://viem.sh/) as well. Viem contains helpful abstractions and modules that will come in handy when using Account Kit. Additionally, many Account Kit modules use `viem` themselves.
 
 :::
 
-## A Simple Light Account Example
+Make sure your new `package.json` file and `tsconfig.json` files look like the following. Note that we added `"type": module` in `package.json` for this example:
 
-Using the SDK, we'll deploy a Light Account and send a User Operation from it. The Light Account will be owned by an Externally Owned Account (EOA). Here's a demonstration:
+::: code-group
 
-<<< @/snippets/light-account.ts
+```json [package.json]
```

Comment:
why are we recommending tsconfig and package.json for peeps? if our library doesn't work with a range of tsconfigs then we broke something I think. 

the package.json is confusing here too... we already told them to `yarn init` and `yarn add`? so this is a given

Reply:
You need to add "type: module" still

Reply:
I did change the npm flag to be `npm init es6 -y` so it's there by default. I just think we should be super clear on each step so devs get unblocked, accordingly to feedback from Ava.